frontend/src/components/pages/TourDeFoos/TourDeFoos.tsx

```diff
@@ -13,49 +14,51 @@ import {HomeMenu} from "../HomeMenu/HomeMenu";
-            contrastText: "#E3E152",
-        },
+  palette: {
+    primary: {
+      main: grey[800],
+      contrastText: "#E3E152",
     },
+  },
 });
 
 function TourDeFoos() {
-    return (
-        <Router>
-            <MuiThemeProvider theme={theme}>
-                <Centered>
-                    <Switch>
-                        <Route path={PATH_FOR_LOGIN_VIEW} exact>
-                            <Button>PATH_FOR_LOGIN_VIEW</Button>
-                        </Route>
-                        <Route path={PATH_FOR_TOURNAMENTS_SELECTION_VIEW} exact>
-                            PATH_FOR_TOURNAMENTS_SELECTION_VIEW
-                        </Route>
-                        <Route path={PATH_FOR_CREATING_NEW_TOURNAMENT_VIEW} exact>
-                            PATH_FOR_CREATING_NEW_TOURNAMENT_VIEW
-                        </Route>
-                        <Route path={PATH_FOR_TOURNAMENT_REGISTRATIONS_VIEW + `/:tournamentId`}>
-                            <Centered>
-                                <TournamentRegistrations/>
-                            </Centered>
-                        </Route>
-                        <Route path={PATH_FOR_PLAYERS_PROFILES_VIEW} exact>
-                            PATH_FOR_PLAYERS_PROFILES_VIEW
-                        </Route>
-                        <Route path={PATH_FOR_PLAYER_PROFILE_CREATION_VIEW} exact>
-                            PATH_FOR_PLAYER_PROFILE_CREATION_VIEW
-                        </Route>
+  return (
+    <Router>
+      <MuiThemeProvider theme={theme}>
+        <Centered>
+          <Switch>
+            <Route path={PATH_FOR_LOGIN_VIEW} exact>
+              <Button>PATH_FOR_LOGIN_VIEW</Button>
+            </Route>
+            <Route path={PATH_FOR_TOURNAMENTS_SELECTION_VIEW} exact>
+              PATH_FOR_TOURNAMENTS_SELECTION_VIEW
+            </Route>
+            <Route path={PATH_FOR_CREATING_NEW_TOURNAMENT_VIEW} exact>
+              PATH_FOR_CREATING_NEW_TOURNAMENT_VIEW
+            </Route>
+            <Route
+              path={PATH_FOR_TOURNAMENT_REGISTRATIONS_VIEW + `/:tournamentId`}
+            >
+              <Centered>
+                <TournamentRegistrations />
+              </Centered>
+            </Route>
+            <Route path={PATH_FOR_PLAYERS_PROFILES_VIEW} exact>
+              PATH_FOR_PLAYERS_PROFILES_VIEW
+            </Route>
+            <Route path={PATH_FOR_PLAYER_PROFILE_CREATION_VIEW} exact>
+              PATH_FOR_PLAYER_PROFILE_CREATION_VIEW
+            </Route>
 
-                        <Route path={PATH_FOR_HOME_VIEW} exact>
-                            <HomeMenu/>
-                        </Route>
-                    </Switch>
-                </Centered>
-            </MuiThemeProvider>
-        </Router>
-    );
+            <Route path={PATH_FOR_HOME_VIEW} exact>
+              <HomeMenu />
+            </Route>
+          </Switch>
+        </Centered>
+      </MuiThemeProvider>
+    </Router>
+  );
```

Comment:
Popraw formatowanie, bo coś konflikty wyczuwam :)

Reply:
no właśnie to już chyba jest poprawione przez npm run format :-)